src/components/BookItems.js

```diff
@@ -0,0 +1,131 @@
+// import React from 'react';
+// import PropTypes from 'prop-types';
+// import '../styles/BookState.css';
+// import { useDispatch } from 'react-redux';
+// import { removeBook } from '../redux/books/booksSlice';
+
+// const BookItems = ({ bookItem }) => {
+//   const {
+//    category, title, author, percentage, chapter, itemId,
+//   } = bookItem;
+//   // const itemIds = bookItem.item_id;
+//   const dispatch = useDispatch();
+
+//   return (
+//     <>
+//       <div className="containerRender" style={{ display: 'flex' }}>
+//         <div className="renderLeft">
+//           <p>{category}</p>
+//           <h3>{title}</h3>
+//           <p>{author}</p>
+//           <div className="leftButton">
+//             <button className="leftButton" type="button">Comment</button>
+//             <button
+//               className="leftButton"
+//               type="button"
+//               onClick={() => {
+//                 dispatch(removeBook(itemId));
+//               }}
+//             >
+//               Remove
+//             </button>
+//             <button className="leftButton" type="button">Edit</button>
+//           </div>
+//         </div>
+//       </div>
+//       <div className="renderRight">
+//         <div className="middle">
+//           <p className="Amid">Progress</p>
```

Comment:
- [ ] please remove any commented-out code, it makes your code look unorganised and harder to read.